keyboards/checkerboards/quark_squared/info.json

```
@@ -1,13 +1,19 @@
 {
     "keyboard_name": "Quark_Squared",
-    "url": "",
+    "url": "https://www.github.com/qmk/qmk_firmware/tree/master/keyboards/checkerboards/quark_squared",
     "maintainer": "nasp",
     "layouts": {
-        "LAYOUT_ortho_2x225u": {
+        "LAYOUT_4_2x225u": {
```

Something like this would be better: 
```suggestion
        "LAYOUT_ortho_4x12_2x225u": {
```